_model-utils/getModel.md

```diff
@@ -0,0 +1,12 @@
+---
+header: getModel
+example: ModelUtils.getModel(path, [attributes], [options], [callback])
+---
+
+Factory method for models, given the models path this will return an instance of that model.  Sets the given `attributes` and `options` on the model. 
```

Comment:
Factory method for models.  Given the model's path this will return an instance of that model with the given `attributes` and `options` set on it.